df.graphics.xml

```diff
@@ -190,6 +190,19 @@
         <pointer name='edge_biome_se' type-name='int32_t' is-array='true'/>
     </struct-type>
 
+    <struct-type type-name='cached_texturest'>
+        <int32_t name='w'/>
+        <int32_t name='h'/>
+        <pointer name='tex'/>
+        <uint8_t name='tex_n'/>
```

Comment:
tag comes before the variant representation, not after

Reply:
nevermind, i'm wrong, this is correct except that the tag should be `int8_t` as `-1` is always a valid value for the union tag field